### PR TITLE
HTTP header with authentication data is called "Authorization"

### PR DESCRIPTION
**Pull request by djmitche**
..not "Authentication".

### CODE REVIEW

**Comment by djmitche** [comment]
@petemoore - do we need to worry about this with the existing mapper stuff?  I can include a backward-compatibility shim to still accept "Authorization" if so, but if those tokens aren't actively being used I'd rather just change the mapper client at the same time.

**Comment by djmitche** [comment]
I added the backward compatibility, just to be safe.  So this isn't quite as much a "breaking change" as originally advertised.

**Comment by Callek** [comment]
Needs a test to exercise the backwards compat functionality. (Thats the reason coverage fell)

File: relengapi/blueprints/tokenauth/test_loader.py [hunk 1]
```diff
@@ -16,28 +16,39 @@
...
     tok = FakeSerializer.prm(1)
     auth = json.loads(
         client.get('/test_tokenauth',
-                   headers=[('Authentication', 'Bearer ' + tok)]).data)
+                   headers=[('Authorization', 'Bearer ' + tok)]).data)
+    eq_(auth['permissions'], ['test_tokenauth.zig'], auth)
+
+
+@test_context.specialize(db_setup=insert_prm)
+def test_loader_good_header_Authentication(app, client):
```
Comment (djmitche) [inline relengapi/blueprints/tokenauth/test_loader.py]: Whoops, I wrote the test but used the new header :)

**Comment by djmitche** [comment]
fixed

**Comment by djmitche** [comment]
That was the only comment, so I'm assuming this is mergeable.

**Comment by Callek** [comment]
:+1:

**Pull request merged**